scale/product/models.py

```diff
@@ -462,6 +516,33 @@ def upload_files(self, file_entries, input_file_ids, job_exe, workspace):
                         product.meta_data = props
                     product.center_point = geo_utils.get_center_point(geom)
 
+            # Add recipe info to product if available.
+            recipe_check = Recipe.objects.get_recipe_for_job(job_exe.job_id)
+            if recipe_check:
+                recipe_manager = RecipeManager()
+                recipe_info = recipe_manager.get_details(recipe_check.id)
```

Comment:
Instead of creating RecipeManager you should use Recipe.objects so you'd do Recipe.objects.get_details().
It's not necessary to call get_details(). Instead I would re-work get_recipe_for_job() to return the RecipeJob model instead of its recipe model (get_recipe_for_job() is only called in one other place). Then we can use recipe_job.recipe, recipe_job.recipe.recipe_type, and recipe_job.job_name to populate product.recipe, product.recipe_type, and product.recipe_job.